README.md

```diff
@@ -30,15 +26,9 @@ Foldernya dinamai seperti `N-url`, di mana `N` adalah angka untuk tujuan penguru
 
 Tipe material didefinisikan oleh file di dalam foldernya:
 
-<<<<<<< HEAD
   - `index.md` berdiri untuk satu bab
```

Comment:
kayaknya satunya diubah bro jadi suatu/sebuah , karena translateannya for a chapter bukan for one chapter. imho